noir_stdlib/src/field/mod.nr

```diff
@@ -94,6 +101,10 @@ impl Field {
     // docs:start:to_be_bytes
     pub fn to_be_bytes<let N: u32>(self: Self) -> [u8; N] {
         // docs:end:to_be_bytes
+        static_assert(
+            N <= modulus_le_bytes().len(),
+            "N must be less than or equal to modulus_le_bytes().len()",
+        );
```

Comment:
Can't comment on it directly but one requirement of `to_le_radix` is that the radix is a power of 2.

Reply:
Also greater than 1 and lte 256

Reply:
Ok, I've added those three checks + basic tests that they're triggered. (wip)